hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/test/java/org/apache/hadoop/yarn/client/api/impl/TestTimelineClient.java

```diff
@@ -78,6 +78,7 @@ public void setup() {
     conf.setBoolean(YarnConfiguration.TIMELINE_SERVICE_ENABLED, true);
     conf.setFloat(YarnConfiguration.TIMELINE_SERVICE_VERSION, 1.0f);
     client = createTimelineClient(conf);
+    TimelineConnector.DEFAULT_SOCKET_TIMEOUT = 10;
```

Comment:
Do you see a chance that test could fail due to a quite low socket timeout?

Reply:
well... it is a good question that made me wonder in case of a successful connection how long the KerberosAuthenticator#190 connect is running, and the plot-twist i just realized all of the tests expect timeout here 😅
If you add a some debug point before and after this line we can observe the debug point after the connect will never be triggered, so technically we can also write 1ms here, the tests can not be broken

Reply:
Thanks, in that case LGTM.